README.md

```diff
@@ -88,151 +103,149 @@ personInfoDictionary[klaviyo.KLPersonZipDictKey] = "02215"
-    
-    KLPersonEmailDictKey 
-    KLPersonFirstNameDictKey
-    KLPersonLastNameDictKey
-    KLPersonPhoneNumberDictKey
-    KLPersonTitleDictKey
-    KLPersonOrganizationDictKey
-    KLPersonCityDictKey
-    KLPersonRegionDictKey
-    KLPersonCountryDictKey
-    KLPersonZipDictKey
-    KLEventIDDictKey
-    KLEventValueDictKey
-
-Lastly, cases where you wish to call `trackEvent` with only the eventName parameter and not have it result in anoynmous user tracking, you can use `setUpUserEmail` to configure your user's email address. By calling this once, usually upon application login, Klaviyo can track all subsequent events as tied to the given user. However, you are also free to override this functionality by passing in a customer properties dictionary at any given time:
+Implementing push notifications requires a few additional snippets of code to enable.: 
+1. Registering users for push notifications.
+2. Sending resulting push tokens to Klaviyo.
+3. Handlinge when users attempt to open your push notifications.
 
-```swift
-    Klaviyo.sharedInstance.setUpUserEmail(userEmail: "john.smith@example.com")
-```
-## Sending Push Notifications
-To be able to send push notifications, you must add a few snippets of code to your application. One to register users for push notifications, one that will send resulting push tokens to Klaviyo, and some final snippets to handle when users attempt to open your push notifications.
+### Sending push notifications
 
-Add the below code to your application wherever you would like to prompt users to register for push notifications. This is often included within `application:didFinishLaunchingWithOptions:`, but it can be placed elsewhere as well. Make sure that whenever this code is called that the Klaviyo SDK has been configured and that `setUpUserEmail:` has been called. This is so that Klaviyo can match app tokens with customers.
+1. Add the following code to your application wherever you would like to prompt users to register for push notifications. This is often included within `application:didFinishLaunchingWithOptions:`, but it can be placed elsewhere as well. When this code is called, ensure that the Klaviyo SDK has beenis configured and that `setUpUserEmail:` has beenis called. This enables Klaviyo to match app tokens with profiles in Klaviyocustomers.
 
 ```swift
-import UserNotifications
-
+    import UserNotifications
 ...
 
-let center = UNUserNotificationCenter.current()
-center.delegate = self as? UNUserNotificationCenterDelegate
-let options: UNAuthorizationOptions = [.alert, .sound, .badge, .provisional]
+    let center = UNUserNotificationCenter.current()
+    center.delegate = self as? UNUserNotificationCenterDelegate
+    let options: UNAuthorizationOptions = [.alert, .sound, .badge, .provisional]
```

Comment:
Do we typically add a tab to code samples? I'm seeing that it's consistent across all code samples? I don't have a strong leaning one way or the other but just a nit suggestion to may be keep them all the same.
Ex:
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/118314354/226375121-0cc364e6-743c-4c4e-9583-2f7dbe0c5db5.png">

Reply:
actually I might leave this for now. We can re-visit the indentation later.